salesforcerest/src/org/wso2/ballerina/connectors/salesforcerest/ClientConnector

```diff
@@ -0,0 +1,475 @@
+package org.wso2.ballerina.connectors.salesforcerest;
+
+import ballerina.lang.messages;
+import ballerina.net.uri;
+import org.wso2.ballerina.connectors.oauth2;
+
+@doc:Description("Salesforcerest client connector")
+@doc:Param("accessToken: The accessToken of the salesforce connected app to access the salesforce REST API")
+@doc:Param("clientId: The clientId of the salesforce connected app to access the salesforce REST API")
+@doc:Param("clientSecret: The clientSecret of the salesforce connected app to access the salesforce REST API")
+@doc:Param("refreshToken: The refreshToken of the salesforce connected app to access the salesforce REST API")
+connector ClientConnector(string accessToken, string clientId, string clientSecret, string refreshToken) {
+
+    string refreshEndpoint = "https://login.salesforce.com/services/oauth2/token";
+    string baseEndpoint = "https://ap4.salesforce.com";
```

Comment:
The api instance should be configurable. Get the baseEndpoint or just the instance value from the user